tests/D2L.CodeStyle.TestAnalyzers.Tests/IgnoreAttribute/IgnoreAnalyzerTests.cs

```diff
@@ -37,6 +37,24 @@ class Test {
 		}
 
 		[Test]
+		public void DocumentWithoutTestIgnore_NoDiag() {
+			const string test = @"
+	using System;
+
+	namespace test {
+		[ReflectionSerializer]

```

Comment:
You shouldn't need this. `ReflectionSerializer` is only for DTOs. See my comments above in terms of making sure `Ignore` is on a test method, and making sure the correct `Ignore` is being checked.